src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -95,9 +98,26 @@ public void RunApplication(Application app)
                         // Give user a chance to save (but does not allow cancellation)
                         viewModel.Exit(allowCancel: false);
                     }
+                    else
+                    {
+                        //show a message dialog box with the exception so the user
+                        //can effectively report the issue.
+                        var shortStackTrace = String.Join(Environment.NewLine,e.StackTrace.Split(Environment.NewLine.ToCharArray()).Take(10));
```

Comment:
Since this is pointing to the wiki, we will need to update there how to file issue related to start up failure.

Reply:
yep, I will do that there.